ml-proto/spec/int.ml

```diff
@@ -208,7 +208,19 @@ struct
   let ge_s x y = x >= y
   let ge_u x y = cmp_u x (>=) y
 
-  let of_string = Rep.of_string
+  (* to_string supports unsigned decimals *)
+  let of_string x =
+    let unsigned_decimal x =
+      let l = String.length x in
+      Rep.add
+        (Rep.mul
+          (Rep.of_string (String.sub x 0 (l - 1)))
+          (Rep.of_int 10))
+        (Rep.of_string (String.sub x (l - 1) 1))
```

Comment:
This code silently wraps on overflow (unless it runs out of stack space first ;-)). I think the spec should explicitly diagnose overflow.